pkgs/data/fonts/siji/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "siji-${date}";
+  date = "2016-05-14";
```

Comment:
The [commit](https://github.com/stark/siji/commit/95369afac3e661cb6d3329ade5219992c88688c1) was done on May 13, so better change to `2016-05-13`.